components/Events/EventList.vue

```diff
@@ -21,7 +22,7 @@ export default {
   props: {
     events: {
       type: Array,
-      required: true,
+      required: false,
```

Comment:
karena component `EventList` ini butuh list of event berupa Array, sebaiknya tetap di buat `required` ya